src/main/java/com/microsoft/azure/datalake/store/ADLStoreClient.java

```diff
@@ -228,6 +232,22 @@ public static ADLStoreClient createClient(String accountFQDN, MsiTokenProvider t
         return createClient(accountFQDN, (AccessTokenProvider) tokenProvider);    }
 
 
+	/**
+	 * Set parameters for exponential backoff
+	 * @param maxRetries Maximum number of retry attempts
+	 * @param exponentialRetryInterval Initial retry interval, in milliseconds
+	 * @param exponentialFactor Retry interval is geometrically increased by this factor on each attempt
+	 */
+	public void setExponentialBackoffParameters(int maxRetries, int exponentialRetryInterval, int exponentialFactor) {
```

Comment:
@wheezil We already had a internal pr for doing this but couldnt get to it due to our bandwith. We will accept this one.

You need to do this through "ADLStoreOptions.java", not through client. Have a option for all these three. And under client "setOptions" set these. Thats the current structure which we follow while setting these options in the hadoop driver.

Reply:
OK.  Thanks for the feedback!

Reply:
@wheezil any updates regarding this?